packages/fiori/src/UploadCollectionItem.hbs

```diff
@@ -0,0 +1,35 @@
+{{>include "../../main/src/ListItem.hbs"}}
```

Comment:
Not sure how we can proceed here. We need to import a template from another package
@ilhan007 @pskelin @MapTo0 @fifoosid 
Opinions?
Should we modify the plugin to understand node_modules/ and write here a full @ui5/webcomponents/dist..../ path?